### PR TITLE
Add missing release notes and @since tags

### PR DESCRIPTION
**Pull request by Benestar**
If I didn't miss something those things also got added in 1.4.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 * Renamed `SerializerFactory::newSnaksSerializer` to `SerializerFactory::newSnakListSerializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newClaimDeserializer` to `DeserializerFactory::newStatementDeserializer`, leaving a b/c alias
 * Renamed `DeserializerFactory::newSnaksDeserializer` to `DeserializerFactory::newSnakListDeserializer`, leaving a b/c alias
+* Added `SerializerFactory::newStatementListSerializer`
+* Added `DeserializerFactory::newStatementListDeserializer`
+* Added `DeserializerFactory::newFingerprintDeserializer`
```
Comment (thiemowmde) [inline RELEASE-NOTES.md]: Hm, why is there no FingerprintSerializer?

Comment (Benestar) [inline RELEASE-NOTES.md]: There is, we just don't have it publicly exposed in a Factory.

Comment (thiemowmde) [inline RELEASE-NOTES.md]: To be more precise, why is the deserializer public via the factory but the serializer is not?

Comment (thiemowmde) [inline RELEASE-NOTES.md]: I think both should not be exposed. There currently is no need for it as far as I can see.

File: src/DeserializerFactory.php [hunk 2]
```diff
@@ -55,8 +55,8 @@ public function __construct( Deserializer $dataValueDeserializer, EntityIdParser
 	 */
 	public function newEntityDeserializer() {
 		return new DispatchingDeserializer( array(
-			new ItemDeserializer( $this->newEntityIdDeserializer(), $this->newFingerprintDeserializer(), $this->newStatementListDeserializer(), $this->newSiteLinkDeserializer() ),
-			new PropertyDeserializer( $this->newEntityIdDeserializer(), $this->newFingerprintDeserializer(), $this->newStatementListDeserializer() )
+			new ItemDeserializer( $this->newEntityIdDeserializer(), new FingerprintDeserializer(), $this->newStatementListDeserializer(), $this->newSiteLinkDeserializer() ),
+			new PropertyDeserializer( $this->newEntityIdDeserializer(), new FingerprintDeserializer(), $this->newStatementListDeserializer() )
```
Comment (thiemowmde) [inline src/DeserializerFactory.php]: Please reuse the same object.

**Comment by thiemowmde** [comment]
I checked and I think everything is covered now and ready to be released.

File: src/DeserializerFactory.php (hunk 2 above)
Comment (Benestar) [inline src/DeserializerFactory.php]: Fixed. This issue already existed before though.

Comment (thiemowmde) [inline src/DeserializerFactory.php]: Cool, thanks.

**Pull request merged**